addons/stock/models/stock_rule.py

```diff
@@ -169,11 +169,12 @@ def _get_message_dict(self):
                 suffix += _("<br>If the products are not available in <b>%s</b>, a rule will be triggered to bring products in this location.", source)
             message_dict = {
                 'pull': _(
-                    'When products are needed in <b>%(destination)s</b>, <br/> <b>%(operation)s</b> are created from <b>%(source_location)s</b> to fulfill the need.',
+                     'When products are needed in <b>%(destination)s</b>, <br/> <b>%(operation)s</b> are created from <b>%(source_location)s</b> to fulfill the need.',
```

Comment:
While you're at it, you could replace `<br/>` with `<br>`. Self-closing tags are an XML thing, they don't exist in HTML. We're lucky because HTML is pretty lenient when it comes to parsing errors, so it'll just ignore the unexpected `/`. But as a rule of thumb, unless you're writing XML, you should never use self-closing tags 👍